flow-typed/types/playback-context.js

```diff
@@ -4,5 +4,6 @@ declare type ProviderPlaybackContextOptions = {
   assetFileIds?: string,
   context?: string,
   streamerType?: string,
-  urlType?: string
+  urlType?: string,
+  adapterData?: Object
```

Comment:
```suggestion
  adapterData?: Array<{[key: string]: string}>
```

Reply:
It's an object of an object,s not an array as mentioned on docs.

Reply:
see the api docs - unfourtunatly the jira tickets has invalid sample code

Reply:
or maybe not, LMC

Reply:
```suggestion
  adapterData?: {[key: string]: {value: string, objectType?: string}}
```

Reply:
you are correct, the oroginal docs are invalid apperantly and ticket has correct form :-)

Reply:
I'm not sure it'll be typed as you wrote, I don't think they have a strict structure

Reply:
it's an API - it has, or at least should have, a strict structure